recipe/meta.yaml

```diff
@@ -11,24 +11,21 @@ source:
 
 build:
   skip: true  # [win]
-  number: 3
+  number: 4
 
 requirements:
   build:
     - {{ compiler('c') }}
+    - {{ stdlib('c') }}
     - {{ compiler('cxx') }}
     - autoconf
     - automake
+    - gnuconfig
+    - librsvg
```

Comment:
Shouldn't this be in host?

Reply:
Confusingly, for this particular recipe the `rsvg-convert` executable provided by `librsvg` is used during the process, but not the shared library.  The previous migration was failing when `librsvg` was in `host` because it was trying to use `rsvg-convert` for the wrong architecture.

Reply:
Ah, good reason! Can you add that as a comment so that the next time it is obvious?

Reply:
Sorry I missed this reply.  I have made a comment as suggested in the most recent commit.  Hopefully we can merge now!

Reply:
@xhochy any chance this can be merged now?